topostats/default_config.yaml

```diff
@@ -34,7 +34,7 @@ dnatracing:
 plotting:
   run: true # Options : true, false
   save_format: png # Options : see https://matplotlib.org/stable/api/_as_gen/matplotlib.pyplot.savefig.html
-  image_set: core  # Options : all, core
+  image_set: all  # Options : all, core
```

Comment:
Only thing I'd say would be to have this default as `core`? Or is it useful for testing and we can update it after?

Reply:
I agree! Changing that now.